google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -25,6 +25,19 @@ plugin.conflict.error.title=Google Plugin Conflict
 plugin.conflict.error.detail=You are running an old plugin ({0}) that conflicts with the installed Google Cloud Tools for Java plugin.
 plugin.conflict.error.action=Please {0}click here{1} to deactivate the old plugin.
 appengine.name=Google App Engine
+appengine.application.not.exist=An App Engine application does not exist for this project.
+appengine.application.required.deployment=An App Engine application must be created before you can deploy to App Engine.
+appengine.application.create={0}Click here{1} to create an application.
+appengine.application.create.instructions=Please select a region for your App Engine Application. Once you choose it, you will not be able to change it later.
+appengine.application.create.documentation=For more information about App Engine regions, see the {0}documentation{1}.
+appengine.application.create.error.transient=An unexpected error occurred while creating the application. Please try again later.
+appengine.application.create.loading=Creating App Engine application in {0}. This can take up to a minute.
+appengine.application.region.supported.environments=The region {0} supports the following App Engine environments:
+appengine.application.region.supports.standard={0}App Engine Standard{1}
```

Comment:
yep, that's currently part of the instructions in the app engine application creation dialog